dbc_module.rb

```diff
@@ -0,0 +1,31 @@
+module DbcMod
+  attr_reader :age, :phase
+  attr_accessor :name
+
+  def initialize(options = {})
+    @phase = 0
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+  end
+
+  def offer_high_five
+    "High five!"
+  end
+
```

Comment:
This method can go into a separate module called "HighFieveable"

Usually for modules that are going to be included in classes, we use "-able" names, so here it'd be HighFiveable.